protocol/avdl/keybase1/user.avdl

```diff
@@ -142,6 +142,36 @@ protocol user {
 
   void uploadUserAvatar(string filename, union { null, ImageCropRect } crop);
 
+  ProfileProofSuggestionsRes profileProofSuggestions(int sessionID);
+  record ProfileProofSuggestionsRes{
+    array<ProfileProofSuggestion> suggestions;
+    boolean showMore;
```

Comment:
should this just be a `belowFold` flag on each suggestion so click `showMore` doesn't result in another RPC call (that will call the same helper function with a full load/sort etc)